### PR TITLE
SCC articles: add metadata

### PR DESCRIPTION
**Pull request by taroth21**
### PR creator: Description

Add metadata from 'Product Docs Metadata per deliverable' spreadsheet. 
Added a stub revhistory based on docserv-config logs.

### PR creator: Are there any relevant issues/feature requests?

Link to our [internal page](https://confluence.suse.com/display/documentation/Adding+metadata+to+all+SUSE+documentation).

### CODE REVIEW

File: system-uptime-tracking/xml/art_system-uptime-tracking.xml [hunk 1]
```diff
@@ -39,6 +40,23 @@
...
+    <meta name="title" its:translate="yes">System Uptime Tracking</meta>
+    <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+    <meta name="description" its:translate="yes">How to monitor the uptime of a client system</meta>
+    <meta name="social-descr" its:translate="yes">Monitoring the uptime of a client system</meta>
+    <meta name="task" its:translate="no">
+      <phrase>Subscription Management</phrase>
+    </meta>
+    <revhistory xml:id="rh-article-system-uptime-tracking">
+      <revision>
+        <date>???</date>
```
Comment (taroth21) [inline system-uptime-tracking/xml/art_system-uptime-tracking.xml:52]: @tbazant : This needs to be filled in whenever the article will be published on d.s.c (for now, it's only published as preview on sd.gh.io for our beta customers)

Comment (tbazant) [inline system-uptime-tracking/xml/art_system-uptime-tracking.xml:52]:
```suggestion
        <date>2024-07-06</date>
```

**Review by tbazant — CHANGES_REQUESTED** [review]
date updated

**Review by janajaeger — APPROVED** [review]
LGTM

**Pull request merged**